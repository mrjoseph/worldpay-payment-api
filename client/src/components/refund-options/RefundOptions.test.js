import React from 'react';
import renderer from 'react-test-renderer';
import RefundOptions from './Refind-options';

describe('Header', () => {
  it('Should render the checkout view without props', () => {
    const rendered = renderer.create(<RefundOptions />);
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
