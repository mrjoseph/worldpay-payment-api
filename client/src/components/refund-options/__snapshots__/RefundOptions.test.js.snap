// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render the checkout view without props 1`] = `
<div
  className="settled-payment-options"
>
  <h3>
    Refund options
  </h3>
  <button
    className="btn btn-primary"
    type="button"
  >
    Refund
  </button>
  <button
    className="btn btn-secondary"
    type="button"
  >
    Partial refund
  </button>
</div>
`;
